cocotb/share/lib/vhpi/VhpiImpl.h

```diff
@@ -207,7 +207,7 @@ class VhpiSignalObjHdl : public GpiSignalObjHdl {
     virtual int initialise(std::string &name, std::string &fq_name);
 
 protected:
-    const vhpiEnumT chr2vhpi(const char value);
+    vhpiEnumT chr2vhpi(const char value);
```

Comment:
```suggestion
    vhpiEnumT chr2vhpi(char value);
```
The other const is meaningless too